src/lib.rs

```diff
@@ -522,6 +574,8 @@ pub mod pallet {
 
```

Comment:
`get_supersig_id_from_index` is only used once in the pallet. And it a 1 line long function.
Just get rid of it and inline it's content where it is used

Reply:
`is_user_in_supersig` should be implemented onto the SuperSig struct. Not taking index as argument.
With this impl if a supersig does not exist this function say that the user is not a member of this supersig. That doesn't exist. Kinda messy. We should first have a supersig, then test if an user is part of it. Not both as the same time, ignoring the `None` case